json/json4s/src/test/scala/sttp/client4/BackendStubJson4sTests.scala

```diff
@@ -0,0 +1,37 @@
+package sttp.client4
+
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.flatspec.AnyFlatSpec
+import org.scalatest.matchers.should.Matchers
+import sttp.client4.testing.SyncBackendStub
+import sttp.model.Uri
+import org.json4s.{native, DefaultFormats, JField, JObject}
+import org.json4s.JsonAST.JString
+
+case class Person(name: String)
+
+class BackendStubJson4sTests extends AnyFlatSpec with Matchers with ScalaFutures {
+
+  implicit val serialization = native.Serialization
+  implicit val formats = DefaultFormats
+
+  import json4s._
+
+  it should "deserialize to json using a string stub" in {
+    val backend = SyncBackendStub.whenAnyRequest.thenRespond("""{"name": "John"}""")
+    val r = basicRequest.get(Uri("http://example.org")).response(asJson[Person]).send(backend)
+
+    r.is200 should be(true)
+    r.body should be(Right(Person("John")))
+  }
+
+  it should "serialize from JsObject using implicit upickleBodySerializer" in {
```

Comment:
The test name should be "serialize from JObject using implicit json4sBodySerializer"